Tapjoy/tpe_prebid_service
# [NGS-730] Honeycomb Audit

## Context

We need to revisit HC implementation and reduce the number of events if possible
non-verbose events doesn’t have much value, convert them to verbose if we can
decrease the number of events if possible, cut some unnecessary reporting

Bump Otel Versions
Update Verbosity Propogation

 ## How To Test

`make build` and `./validate.sh`
end to end testing
Ensure verbosity for DSP events are correctly propagated 

 ## How Has This Been Tested?

`make build` and `./validate.sh`
end to end testing

 ## Jira Link

Jira: https://tapjoy.atlassian.net/jira/software/projects/NGS/boards/65?selectedIssue=NGS-730&text=honeycomb

 ## Related PRs
MBS: https://github.com/Tapjoy/mediation_bid_service/pull/1260
TPE: https://github.com/Tapjoy/tpe_prebid_service/pull/143
